lib/database/migrations/20240119141430-create-simulation-passes.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+    up: async (queryInterface, Sequelize) => queryInterface.sequelize.transaction(async (transaction) => {
+        await queryInterface.createTable('simulation_passes', {
+            id: {
+                type: Sequelize.INTEGER,
+                allowNull: false,
+                primaryKey: true,
+                autoIncrement: true,
+            },
+            name: {
+                type: Sequelize.STRING,
+                unique: true,
+            },
+            description: {
+                type: Sequelize.TEXT,
+            },
+            jira_id: {
+                type: Sequelize.STRING,
+            },
+            pwg: {
```

Comment:
Is this a well known acronym?
if it is, please put its explicit version in comment in the model (and maybe even as comment in table column)

Reply:
yes `Physicist working group`, comment added